### ISSUE TITLE
Recent update causing issues with macOS.

### ISSUE THREAD

**Issue by GitStoph** [firehol/netdata#1903]
recently updated NetData on a couple caching servers I have. ran into some interesting issues.
- the http://my.fqdn:19999 stopped working, citing that it was an access issue.
   I went through and verified permissions were correct throughout and they were. 
   `sudo chown -R user:group /Users/myuser/opt/netdata` at the very root folder didn't fix anything even.
   After doing a full uninstall and reinstall, I realized that the netdata.conf file looks new/different. Now there's a new [web] section. I manually edited the 
```
[web]
         web files owner = myuser
         web files group = mygroup
```
and restarted the netdata process. I now finally had access to the page.

I went to another macOS server that wasn't completely reinstalled, just updated. I found and edited the same lines (on the old style conf) and bam, that one was now working as well. Just know that these need to be manually edited now to view the netdata page. **NOTE** Netdata does continue to run/work in the background. Custom html's drawing data/charts from it will still function, and prometheus will still scrape successfully, one just can't access the main http://my.machine:19999 page in a browser. 

I haven't had time to troubleshoot yet, but I've noticed that the Registry also stopped working somewhere along the line. Unsure what that's about. Will follow up if I find anything there.

**Comment by ktsaou** [comment]
Hi, I am very sorry for this. It shouldn't happen.

You are saying that you had:

```
[global]
         web files owner = myuser
         web files group = mygroup
```

and the latest netdata did not move them by itself to:

```
[web]
         web files owner = myuser
         web files group = mygroup
```

It is supposed to do this by itself. Do you have one of your old configs? I would like to have your `[global]` section to reproduce this.

**Comment by GitStoph** [comment]
Yessir. Also, got the registry nailed down. Using Google Chrome and allowed exceptions for each netdata instance but the browser didn't honor them apparently. Allowing any and all cookies and visiting each then finally filled the registry. Just thought it odd since the exceptions were made.. but whatever, it's solved there. 
old:
```
[global]
        # glibc malloc arena max for plugins = 1
        # config directory = /Users/myuser/opt/netdata/etc/netdata
        # plugins directory = /Users/myuser/opt/netdata/usr/libexec/netdata/plugins.d
        # web files directory = /Users/myuser/opt/netdata/usr/share/netdata/web
        # cache directory = /Users/myuser/opt/netdata/var/cache/netdata
        # lib directory = /Users/myuser/opt/netdata/var/lib/netdata
        # log directory = /Users/myuser/opt/netdata/var/log/netdata
        # host access prefix =
        # home directory = /Users/myuser/opt/netdata/var/cache/netdata
        # debug flags = 0x00000000
        # debug log = /Users/myuser/opt/netdata/var/log/netdata/debug.log
        # error log = /Users/myuser/opt/netdata/var/log/netdata/error.log
        # access log = /Users/myuser/opt/netdata/var/log/netdata/access.log
        # errors flood protection period = 1200
        # errors to trigger flood protection = 200
        # memory mode = save
        # hostname = MyHostName
        # history = 3600
        # update every = 1
        # pthread stack size = 524288
        # run as user = netdata
         web files owner = myuser
         web files group = mygroup
        # http port listen backlog = 100
        # default port = 19999
        # bind to = *
        # multi threaded web server = yes
        # disconnect idle web clients after seconds = 60
        # respect web browser do not track policy = no
        # enable web responses gzip compression = yes
        # web compression strategy = default
        # web compression level = 3
```
New:
```
[global]
        # glibc malloc arena max for plugins = 1
        # hostname = myhostname
        # history = 3996
        # update every = 1
        # config directory = /Users/myuser/opt/netdata/etc/netdata
        # log directory = /Users/myuser/opt/netdata/var/log/netdata
        # plugins directory = /Users/myuser/opt/netdata/usr/libexec/netdata/plugins.d
        # web files directory = /Users/myuser/opt/netdata/usr/share/netdata/web
        # cache directory = /Users/myuser/opt/netdata/var/cache/netdata
        # lib directory = /Users/myuser/opt/netdata/var/lib/netdata
        # home directory = /Users/myuser/opt/netdata/var/cache/netdata
        # memory mode = save
        # host access prefix =
        # debug flags = 0x0000000000000000
        # debug log = /Users/myuser/opt/netdata/var/log/netdata/debug.log
        # error log = /Users/myuser/opt/netdata/var/log/netdata/error.log
        # access log = /Users/myuser/opt/netdata/var/log/netdata/access.log
        # errors flood protection period = 1200
        # errors to trigger flood protection = 200
        # pthread stack size = 524288
        # run as user = netdata
        # OOM score = 1000
        # process nice level = 19
        # cleanup obsolete charts after seconds = 3600
        # delete obsolete charts files = yes
        # delete orphan hosts files = yes

[web]
         web files owner = myuser
         web files group = mygroup
        # listen backlog = 100
        # default port = 19999
        # bind to = *
        # mode = multi-threaded
        # disconnect idle clients after seconds = 60
        # respect do not track policy = no
        # x-frame-options response header =
        # enable gzip compression = yes
        # gzip compression strategy = default
        # gzip compression level = 3
```

**Comment by ktsaou** [comment]
hm... just tried that and it works. Try it yourself too:

1. Move the 2 settings under `[global]`.
2. Restart netdata
3. Fetch `http://your.netdata.ip:19999/netdata.conf`
4. The settings appear in the `[web]` section

Something else changed. Any ideas what that could be?

**Comment by GitStoph** [comment]
Sorry, didn't see your reply until now.

Previously I never had those two sections in the conf file configured, they were always just commented out on OSX. With the update something changed that now required them to be uncommented and configured correctly. 

> Move the 2 settings under [global].

On 3 of my 4 caching servers I am using the old version of the conf file with success if that's what you were referring to? They did not need to be updated to the new conf file, just now need that section configured. I know i'm talking in circles, but does that make sense?

**Comment by GitStoph** [comment]
Mostly I wanted to make a searchable note for others that if they're running netdata on OSX and didn't previously have this part of their conf configured, they'll now need to if viewing the main 19999 page is something they like to do. Pulling data to custom dashboards was still working.

**Comment by ktsaou** [comment]
Yes, I understood that. I am just trying to figure if there is a way to avoid this for the users that will update in the future. If we could pinpoint the change causing this, I could handle it in the code, so that all the folks that will update their netdata later, will not face the issue at all...

**Comment by GitStoph** [comment]
Understood. Is there a way I can see the previous build, would that info help? All I did for the update process was cd into ~/opt/netdata and run the `sudo ./netdata-updater.sh`, and notice the issue. After seeing at the web page that I didnt have permissions to view the website, I chowned the affected folder and worked my way up to the root netdata folder. That's when I figured out it was a conf file setting.
```
curUser=`python -c 'from SystemConfiguration import SCDynamicStoreCopyConsoleUser; import sys; username = (SCDynamicStoreCopyConsoleUser(None, None, None) or [None])[0]; username = [username,""][username in [u"loginwindow", None, u""]]; sys.stdout.write(username + "\n");'`
```
I know the following variable can get you the currently logged in user on a Mac. Perhaps that could be integrated with another variable that grabs group membership and prints it into the conf file?
